django/db/models/query.py

```diff
@@ -647,6 +647,7 @@ def create(self, **kwargs):
         """
         obj = self.model(**kwargs)
         self._for_write = True
+        obj.for_create = True
```

Comment:
Unfortunately, we cannot do this as it would be backward incompatible for all models with `for_create` fields, methods, or properties.